app/components/Application/style.less

```diff
@@ -11,7 +11,7 @@ div.topNav {
   position: fixed;
   width: 100%;
   top: 0;
-  z-index: 10;
+  z-index: 9;
```

Comment:
@RonaldoLee 

The sideNav z-index is 10, topNav z-index should be smaller than sideNav. So I changed here.